### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/security/simple-oauth2.md`

### PR DESCRIPTION
**Pull request by LidiaDomingos**
Hi!! I would like to contribute with portuguese translations for the security part!

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

**Review by cassiobotaro — APPROVED** [review]

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+
+Cada “scope” é apenas uma string (sem espaços).
+
+Normalmente são usados ​​para declarar permissões de segurança específicas, por exemplo:
+
+* `users:read` ou `users:write` são exemplos comuns.
+* `instagram_basic` é usado pelo Facebook e Instagram.
+* `https://www.googleapis.com/auth/drive` é usado pelo Google.
+
+/// info | "Informação"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:35]: You can remove the quotes here:

```
/// info |  Informação
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+
+OAuth2 especifica que ao usar o "password flow" (fluxo de senha), que estamos usando, o cliente/usuário deve enviar os campos `username` e `password` como dados do formulário.
+
+E a especificação diz que os campos devem ser nomeados assim. Portanto, `user-name` ou `email` não funcionariam.
+
+Mas não se preocupe, você pode mostrá-lo como quiser aos usuários finais no frontend.
+
+E seus modelos de banco de dados podem usar qualquer outro nome que você desejar.
+
+Mas para a *path operation* (operação de caminho) de login, precisamos usar esses nomes para serem compatíveis com a especificação (e poder, por exemplo, usar o sistema integrado de documentação da API).
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:17]: you can translate *path operation* to *operação de rota*

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 3]
```diff
@@ -0,0 +1,541 @@
...
+
+///
+
+## Código para conseguir o `username` e a `password`
+
+Agora vamos usar os utilitários fornecidos pelo **FastAPI** para lidar com isso.
+
+### `OAuth2PasswordRequestForm`
+
+Primeiro, importe `OAuth2PasswordRequestForm` e use-o como uma dependência com `Depends` na *path operation* (operação de caminho) para `/token`:
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:53]: you can translate path operation to operação de rota

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="4  79"
+{!> ../../docs_src/security/tutorial003_an.py!}
+```
+
+////
+
+//// tab | Python 3.10+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:81]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 5]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="2  74"
+{!> ../../docs_src/security/tutorial003_py310.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:95]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 6]
```diff
@@ -0,0 +1,541 @@
...
+////
+
+`OAuth2PasswordRequestForm` é uma dependência de classe que declara um corpo de formulário com:
+
+* O `username`.
+* A `password`.
+* Um campo `scope` opcional como uma string grande, composta de strings separadas por espaços.
+* Um `grant_type` (tipo de concessão) opcional.
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:114]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 7]
```diff
@@ -0,0 +1,541 @@
...
+A especificação OAuth2 na verdade *requer* um campo `grant_type` com um valor fixo de `password`, mas `OAuth2PasswordRequestForm` não o impõe.
+
+Se você precisar aplicá-lo, use `OAuth2PasswordRequestFormStrict` em vez de `OAuth2PasswordRequestForm`.
+
+///
+
+* Um `client_id` opcional (não precisamos dele em nosso exemplo).
+* Um `client_secret` opcional (não precisamos dele em nosso exemplo).
+
+/// info | "Informação"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:125]: you can remove the quotes here to:

```
/// info | Informação
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 8]
```diff
@@ -0,0 +1,541 @@
...
+
+Mas `OAuth2PasswordRequestForm` é apenas uma dependência de classe que você mesmo poderia ter escrito ou poderia ter declarado os parâmetros do `Form` (formulário) diretamente.
+
+Mas como é um caso de uso comum, ele é fornecido diretamente pelo **FastAPI**, apenas para facilitar.
+
+///
+
+### Use os dados do formulário
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:139]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 9]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="3  80-82"
+{!> ../../docs_src/security/tutorial003_an.py!}
+```
+
+////
+
+//// tab | Python 3.10+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:179]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 10]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="1  75-77"
+{!> ../../docs_src/security/tutorial003_py310.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:193]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 11]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="83-86"
+{!> ../../docs_src/security/tutorial003_an.py!}
+```
+
+////
+
+//// tab | Python 3.10+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:255]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 12]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="78-81"
+{!> ../../docs_src/security/tutorial003_py310.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:269]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 13]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="80-83"
+{!> ../../docs_src/security/tutorial003.py!}
+```
+
+////
+
+#### Sobre `**user_dict`
+
+`UserInDB(**user_dict)` means:
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:283]: I think you forgot to remove this line

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 14]
```diff
@@ -0,0 +1,541 @@
...
+
+A resposta do endpoint `token` deve ser um objeto JSON.
+
+Deve ter um `token_type`. No nosso caso, como estamos usando tokens "Bearer", o tipo de token deve ser "`bearer`".
+
+E deve ter um `access_token`, com uma string contendo nosso token de acesso.
+
+Para este exemplo simples, seremos completamente inseguros e retornaremos o mesmo `username` do token.
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:315]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 15]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="88"
+{!> ../../docs_src/security/tutorial003_an.py!}
+```
+
+////
+
+//// tab | Python 3.10+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:349]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 16]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="83"
+{!> ../../docs_src/security/tutorial003_py310.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:363]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 17]
```diff
@@ -0,0 +1,541 @@
...
+
+///
+
+```Python hl_lines="85"
+{!> ../../docs_src/security/tutorial003.py!}
+```
+
+////
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:375]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 18]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="59-67  70-75  95"
+{!> ../../docs_src/security/tutorial003_an.py!}
+```
+
+////
+
+//// tab | Python 3.10+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:425]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 19]
```diff
@@ -0,0 +1,541 @@
...
+
+```Python hl_lines="56-64  67-70  88"
+{!> ../../docs_src/security/tutorial003_py310.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip | "Dica"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:439]: you can remove the quotes here to:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/security/simple-oauth2.md [hunk 20]
```diff
@@ -0,0 +1,541 @@
...
+
+///
+
+```Python hl_lines="58-66  69-72  90"
+{!> ../../docs_src/security/tutorial003.py!}
+```
+
+////
+
+/// info | "Informação"
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/security/simple-oauth2.md:451]: you can remove the quotes here to:

```
/// info | Informação
```

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great work \o/

just one line that needs to be removed (283)

The other comments you can change if you want (it would be nice to keep the standards)

**Review by ceb10n — APPROVED** [review]
Thank you ✨

**Comment by alejsdev** [comment]
Thanks for the contribution! @LidiaDomingos :nerd_face:

**Pull request merged**